pom.xml

```diff
@@ -110,6 +110,11 @@
             <version>1.7.30</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.exasol</groupId>
+            <artifactId>database-cleaner</artifactId>
+            <version>1.0.0</version>
+        </dependency>
```

Comment:
Does it need compile scope?

The readme indicates it as compile time dependency.

Reply:
yes. I think default is compile scope